docs/build_release.md

```diff
@@ -1,46 +1,34 @@
 # Presidio build and release
 
 Presidio continuous processes govern its integrity and stability through the dev, test and and release phases.
-The project currently supports [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) using YAML pipelines which can be easily imported to any Azure Pipelines instance.<br/>
+The project currently supports [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) using YAML pipelines which can be easily imported to any Azure Pipelines instance.
 
 ## Presidio Azure Devops Pipelines
 
-Azure Pipelines [YAML Schema](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema) allows for code reuse using [templates](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops).<br/>
+Azure Pipelines [templates](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops) allows for code reuse using [YAML Schema](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema).
 
-***[Presidio Build and Push template](../pipelines/templates/presidio-build-template.yaml)*** - build, test and push presidio images
+***[Presidio CI Pipeline](../pipelines/CI-presidio.yaml)*** - is the pipeline which is used to buil, test and  publish persidio services images.
 
-- Push and publish steps only taken when build is not triggered by a PR.
-- Parameters:
-    - registry_parameter - the name of container registry service connection
-    - registry_name_parameter - the full name of the registry login
-    - deps_label_parameter - the base image (deps) label to be used for container build
+***[Presidio Build and Push template](../pipelines/templates/build-test-publish.yaml)*** - is the stages-template which contains the build, test and push logic of presidio. There are four stages for the build:
```

Comment:
The difference between push and deploy is a bit confusing (maybe just to me)

Reply:
replaced push with publish. deploy has no reference in this doc as we are currently not automating it so it irrelevant.